lib/core.js

```diff
@@ -725,4 +759,18 @@ function closeAll (...storages) {
   })
 }
 
+function createCompatManifest (publicKey) {
+  return {
+    hash: 'blake2b',
+    namespace: b4a.alloc(32),
```

Comment:
if you leave `namespace` and `entropy` undefined here, we can handle them when creating the auth